docs/application/dotnet/guides/app-management/widget-control.md

```diff
@@ -0,0 +1,151 @@
+
+    ```
+    WidgetControl control = new WidgetControl(Your Widget ID);
+    ```
+
+2.  Operate on the control:
+    -   Get the main application ID, package ID, and available scale lists from the control:
+
+        ```
+        string mainId = control.MainAppId;
+        string packageId = control.PackageId;
+        IEnumerable<WidgetControl.Scale> scales = control.GetScales();
+        ```
+
+
+<a name="listening_events"></a>
+## Listening Widget Lifecycle Events on Widget Applications
+
+To listen these widget lifecycle events, follow the steps:
+
+1.  Create an instance of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class with the ID of the widget application, as shown in the following code:
+
+    ```
+    WidgetControl control = new WidgetControl(Your Widget ID);
+    ```
+
+2.  Operate on the control:
+    -   Add lifecycle listeners on the control:
+
+        ```
+        private static void OnCreated(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+        {
+            string instanceId = args.InstanceId;
+            string widgetId = args.WidgetId;
+            /// Widget application is created
+        }
+
+        private static void OnDestroyed(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+        {
+            /// Widget application is destroyed
+        }
+
+        private static void OnPaused(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+        {
+            /// Widget application is paused
+        }
+
+        private static void OnResumed(object sender, Tizen.Applications.WidgetLifecycleEventArgs args)
+        {
+            /// Widget application is resumed
+        }
+
+        control.Created += OnCreated;
+        control.Created += OnDestroyed;
+        control.Created += OnPaused;
+        control.Created += OnResumed;
+
+        string mainId = control.MainAppId;
+        string packageId = control.PackageId;
+        IEnumerable<WidgetControl.Scale> scales = control.GetScales();
+        ```
+
+<a name="communicating_instances"></a>
+## Communicating with Running Widget Instances
+
+To communicate with these running widget instances, follow the steps:
+
+1.  Create an instance of the [Tizen.Applications.WidgetControl](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.WidgetControl.html) class with the ID of the widget application, as shown in the following code:
+
+    ```
+    WidgetControl control = new WidgetControl(Your Widget ID);
+    ```
+
+2.  Operate on the control:
+    -   Get running widget instances:
+
+    ```
+    IEnumerable<WidgetControl.Instance> instances = control.GetInstances();
+    ```
+
+3.  Operate on the instances:
+    -   Get details of running widget instances and send an update to widget application:
+
+    ```
```

Comment:
Please check the indentation of sample code.

Reply:
Done